4.analyze_data/README.md

```diff
@@ -2,10 +2,10 @@
 
 In this module, we perform analysis on the data to uncover answers to our goals as specified in [the main README](../README.md).
 
-
 The analyses we perform are:
 
 1. Linear modeling per feature in each plate. The beta coefficients for WT treatment contribution and cell count contribution are plotted as a scatter plot.
```

Comment:
```suggestion
1. Linear modeling per feature in each plate. 
The beta coefficients for WT treatment contribution and cell count contribution are plotted as a scatter plot.
```

Also one sentence per line suggestion here

Reply:
Since this uses the number formatting, adding a new line would cause that sentence to be number 2 even though it is still part of number 1.

I have decided to turn this into one sentence instead.